CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.1.0]
+### Changed
+- Allow deserializing vaults with unrecognized keyrings ([#169](https://github.com/MetaMask/KeyringController/pull/169))
+  - When deserializing a vault with an unrecognized keyring, the controller will no longer crash. The unrecogized keyring vault data will be preserved in the vault for future use, but will otherwise be ignored.
```

Comment:
To confirm, was the unrecognized keyring only found using MV3? If so, could we update this description to indicate this is a non-issue for manifest v2?